### DISCUSSION TITLE
When would be a good time for us all to hang out at District Brew Yards in July or early August?

### DISCUSSION THREAD

**Discussion by mizquierdo2** [Open-Research-Program/HCOE#4] [General]
Are weekends or weekdays better for people and what times?

**Comment by Jdlopez0108** [comment]
Weekday afternoons (after 3 or 4pm or so) work best for me. Not opposed to a weekend time either, though.

**Comment by jasson907** [comment]
Most weekdays would work for me too! After 4 is good - Weekends can also work as well.

**Comment by CharliePerezSuarez** [comment]
After 4:30 weekdays, or weekends work, too

**Comment by ADParedes** [comment]
This would be great!  I hope we can get together in person.  I'm free T,W,TH,Sat, Sun 4pm-10PM.

**Comment by vgodin4** [comment]
I think July meeting would be nice!
Personally I am free after 5! :)

**Comment by bguill3** [comment]
I can also do most weekday evenings and weekends. :)
